### ISSUE TITLE
Predicting on New Data

### ISSUE THREAD

**Issue by mrsirons** [jdb78/pytorch-forecasting#1152]
I am using the TFT model to predict sales volume per sku 5 months in the future (to predict how much stock to order). Thank you so much for this great model!

I have 2 general questions about using the TFT model to predict on new data.
1- When training, I am using 24 months as the encoder length and 5 months as the validations set (for example: June - October). When I predict on 5 future months of data (for example: November - March) is the model taking into account the data from the validation set or only the training data?

2- I want to use this model each month to predict sales 5 months in the future. Can I use the same model to keep predicting from month to month or do I need to retrain the model each month? Is there some time of incremental training I can use?
